build/mac/BUILD.gn

```diff
@@ -344,14 +344,24 @@ copy("updater_parts_script") {
   outputs = [ "$packaging_dir/signing/updater_parts.py" ]
 }
 
+generated_pkg_scripts = "$root_gen_dir/pkg-scripts"
```

Comment:
consider using `$target_gen_dir/pkg-script` to store all files in `gen/brave/build/mac/pkg-scripts`. This will be in line with `//brave/build/mac/pkg-scripts`.

Reply:
Resolved in 8c5d388c9fd272bb2e22716e404aa28e53f99aa6.